.changelog

```diff
@@ -7,3 +7,4 @@
 
 * Added note about reserved characters in Proxy passwords(bsc#1134836)
 * Syntax and presentation updates to Salt Guide
+* Added Bootstrapping with SSH content
```

Comment:
I'd be specific, that this is about SSH key (bootstrapping via SSH (with password) was already supported before).
```suggestion
* Added Bootstrapping with SSH key content
```